src/repository/sessions.rs

```diff
@@ -66,16 +72,19 @@ impl SessionRepository for PgSessionRepository {
 
     #[tracing::instrument(skip_all)]
     async fn get(&self, id: &str) -> Result<Session, SessionGetError> {
-        let session = sqlx::query_as::<_, Session>("
+        info!(id);
+        let session = sqlx::query_as::<_, Session>(
+            "
             SELECT session_id, users.user_id, expires, email, password_hash
             FROM sessions JOIN users
             ON sessions.user_id = users.user_id
             WHERE sessions.session_id = $1
-        ")
-            .bind(id)
-            .fetch_optional(&self.pool)
-            .await;
-    
+        ",
+        )
+        .bind(id)
+        .fetch_optional(&self.pool)
+        .await;
```

Comment:
Weird indentation?

Reply:
what?

Reply:
I think formatting gets weird when theres a multiline string, because normally chained function calls look like
```
struct
    .f1()
    .f2()
    .f3()
```
But here it's 
```
struct
.f1()
.f2()
.f3()
```